src/instance.ts

```diff
@@ -192,6 +238,249 @@ class Instance extends common.GrpcServiceObject {
     this.databases_ = new Map<string, Database>();
   }
 
+  /**
+   * Get a reference to a Backup object.
+   *
+   * @throws {Error} If any parameter is not provided.
+   *
+   * @param {string} backupId The name of the backup.
+   * @param {string} databasePath the path of the backup.  Only needs to be specified for new backups.
+   * @param {PreciseDate} expireTime expiry time of the backup.  Only needs to be specified for new backups.
```

Comment:
To my knowledge, this would be a new pattern for us, where the accessor method optionally accepts creation-time details. Instead, we would usually wait until the user tells us they want to create the object to collect any required information. So, for this case, the user would only be asked for the name of the backup, then when they call "backup.create()", they would provide `expireTime` then.

Reply:
I agree. Changed to passing the arguments to `create()` instead. Thanks for this suggestion.